tests/FeedItemSourceTest.php

```diff
@@ -216,11 +216,11 @@ public function testNormalize()
                 'name' => FeedItemInput::EXAMPLE_DATA['location']['name'],
                 'mail' => FeedItemInput::EXAMPLE_DATA['location']['mail'],
                 'coordinates' => [
-                    'lat' => FeedItemInput::EXAMPLE_DATA['location']['coordinates']['lat'],
-                    'long' => FeedItemInput::EXAMPLE_DATA['location']['coordinates']['lon'],
+                    'latitude' => FeedItemInput::EXAMPLE_DATA['location']['coordinates']['lat'],
+                    'longitude' => FeedItemInput::EXAMPLE_DATA['location']['coordinates']['lon'],
                 ],
             ],
-            'tags' => [FeedItemInput::EXAMPLE_DATA['tags_string']],
+            'tags' => explode(',', FeedItemInput::EXAMPLE_DATA['tags_string']),
```

Comment:
Shouldn't we expect a list of normalized tags, i.e. `['laserskæring', 'lasercut', '3D print']`, here?

```php
explode(',', FeedItemInput::EXAMPLE_DATA['tags_string']) === ['laserskæring', 'lasercut ', ' 3D print']
```
(cf. https://github.com/itk-dev/event-database-imports/blob/develop/tests/Utils/FeedItemInput.php#L22)

Reply:
No, this is just testing the class used by Valinor, which will only change the list from string to array. The normalization happens later on in the TagNormalizer